### PR TITLE
Add color, icon and defaultVISTheme for workspace

### PR DESCRIPTION
**Pull request by wanglam**
### Description

Add color, icon and defaultVISTheme for workspace

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx [hunk 1]
```diff
@@ -76,6 +85,10 @@ export const WorkspaceForm = ({
 
   const [name, setName] = useState(defaultValues?.name);
   const [description, setDescription] = useState(defaultValues?.description);
+  const [color, setColor] = useState(defaultValues?.color);
+  const [icon, setIcon] = useState(defaultValues?.icon);
+  const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx:90]: I do not know if it would be good to use a object state instead of multiple isolated states in this case.

**Review by SuZhou-Joe — APPROVED** [review]

File: src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx (hunk 1 above)
Comment (wanglam) [reply src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx:90]: For now, we don't know too much relations between these three fields. The defaultVISTheme and icon will be little bit complex. The icon may need to support custom icon upload and defaultVISTheme may contains multiple colors. Not sure the final design for this part.

File: src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx [hunk 2]
```diff
@@ -267,74 +321,67 @@ export const WorkspaceForm = ({
...
+                      ? featureOrGroup.name
+                      : featureOrGroup.id
+                  }
+                  onChange={
+                    isWorkspaceFeatureGroup(featureOrGroup)
+                      ? handleFeatureGroupChange
+                      : handleFeatureCheckboxChange
+                  }
+                  label={`${
+                    featureOrGroup.name === 'OpenSearch Plugins'
```
Comment (ruanyl) [inline src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx:353]: Could you help to revert this change? It's just a hack for demo, we don't need it now.

Comment (wanglam) [reply src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx]: Removed

**Review by ruanyl — APPROVED** [review]
